tests/integration-tests/conftest.py

```diff
@@ -1810,3 +1811,87 @@ def create_efs(num=1):
     if not request.config.getoption("no_delete"):
         for stack in created_stacks:
             cfn_stacks_factory.delete_stack(stack.name, region)
+
+
+@pytest.fixture(scope="class")
+def mount_target_stack_factory(cfn_stacks_factory, request, region, vpc_stack):
```

Comment:
NIT - since this is specific to EFS, I would add this info in the method name

Reply:
Done